runtimes/eden/src/migrations.rs

```diff
@@ -17,85 +17,19 @@ where
-
-		StorageVersion::new(4).put::<pallet_membership::Pallet<T, pallet_membership::pallet::Instance3>>();
-
-		// TechnicalCommittee: pallet_collective::<Instance1>
-		// Found 3 keys (0.19s)
-		// key: 0xed25f63942de25ac5253ba64b5eb64d1ba7fb8745735dc3be2a2c61a72c39e78
-		//      technicalCommittee.members: Vec<AccountId32> list of valid keys.
-		// key: 0xed25f63942de25ac5253ba64b5eb64d16254e9d55588784fa2a62b726696e2b1
-		//      technicalCommittee.proposalCount: u32 = 329
-		// key: 0xed25f63942de25ac5253ba64b5eb64d188c2f7188c6fdd1dffae2fa0d171f440
-		//      technicalCommittee.proposals: Vec<H256> = []
-
-		// Source code unchanged since 2021
-		// 03b294641ef substrate/frame/membership/src/lib.rs (Qinxuan Chen         2021-09-07 20:17:26 +0800  44)  const STORAGE_VERSION: StorageVersion = StorageVersion::new(4);
-		// *** This commit changes from old to new frame macros,
-		// decl_storage!{
-		//    Members get(fn members): Vec<T::AccountId>;
-		// }
-		// changed to:
-		// 	#[pallet::storage]
-		//  #[pallet::getter(fn members)]
-		//  pub type Members<T: Config<I>, I: 'static = ()> = StorageValue<_, Vec<T::AccountId>, ValueQuery>;
-		// *** Migration code only included name change functions.
-
-		// Onchain storage version = 4 in source code - unchanged any new data will be in the v4 format
-
-		StorageVersion::new(4).put::<pallet_collective::Pallet<T, pallet_collective::pallet::Instance1>>();
-
-		// https://github.com/paritytech/substrate/pull/12813
-		// moves funds to inactive if we don't need that this is OK.
-
-		// Onchain storage version = 1 in source code - unchanged any new data will be in the v1 format
-
-		StorageVersion::new(1).put::<pallet_balances::Pallet<T>>();
-
-		// Two keys already migrated.
-		// The call to pallet_xcm::migration::v1::MigrateToV1::<Runtime>::on_runtime_upgrade() fails.
-		// That migration code supposes that the value in the storage is of the old type which is not true,
-		// because two new values of the new type were inserted in the VersionNotifyTargets map which is
-		// the subject of that migration. One of the new values are for Moonbeam which got inserted in
-		// the block 3351853 which is the first block after the parachain restart and the second one is
-		// for Polkadot which got inserted in 3614349 16 days ago. I believe we don’t need this migration.
-		// If in the future there was any issue in any XCM interactions with Moonbeam we can force set the
-		// storage entry for that single value to use proof_size = 65536 (the new default).
-		StorageVersion::new(1).put::<pallet_xcm::Pallet<T>>();
-
 		// Size of onchain storage is 0 safe to upgrade storage version
 		// Onchain storage version = 1 in source code - unchanged any new data will be in the v1 format
 		StorageVersion::new(1).put::<pallet_preimage::Pallet<T>>();
+		// pallet_uniques adding a storage version not chaning anything
+		StorageVersion::new(1).put::<pallet_uniques::Pallet<T>>();
```

Comment:
Are we sure our deployed version is actually 1? No migrations needed?

Reply:
double checking

Reply:
sure